src/libs/kata-types/src/annotations/mod.rs

```diff
@@ -376,6 +378,37 @@ impl Annotation {
         self.get(CONTAINER_TYPE_KEY)
     }
 
+    /// Get the annotation of cpu quota for sandbox
+    pub fn get_sandbox_cpu_quota(&self) -> u64 {
+        let value = self
+            .get_value::<u64>(SANDBOX_CPU_QUOTA_KEY)
+            .unwrap_or(Some(0));
+        if let Some(q) = value {
+            return q;
+        }
+        0
```

Comment:
Nothing wrong with this form, but it would be more idiomatic to `match` on value (here and below).

Reply:
Thanks @jodh-intel , nice catch and fixed by using unwrap_or(0). Btw could you remove the do-not-merge label because the target branch is already main now.

Reply:
> Btw could you remove the do-not-merge label because the target branch is already main now.

Done.

/cc @liubin.